cmd/kubeadm/app/cmd/alpha/kubeconfig.go

```diff
@@ -40,7 +39,7 @@ var (
 
 	userKubeconfigExample = cmdutil.Examples(`
 	# Output a kubeconfig file for an additional user named foo
```

Comment:
probably best to indicate that --config is the kubeadm configuration, as mentioned here:
https://github.com/kubernetes/kubernetes/pull/94879/files#r492257006